src/alert/base.rs

```diff
@@ -284,13 +284,15 @@ pub async fn run_alert_worker<T: AlertWorker>(
                         .map_err(AlertWorkerError::RemoveAlertError)?;
                 }
                 _ => {
-                    warn!(error = %error, "Error processing alert, requeueing");
-                    con.lpush::<&str, Vec<u8>, isize>(&input_queue_name, avro_bytes.clone())
-                        .await
-                        .map_err(AlertWorkerError::PushAlertError)?;
-                    con.lrem::<&str, Vec<u8>, isize>(&temp_queue_name, 1, avro_bytes)
-                        .await
-                        .map_err(AlertWorkerError::RemoveAlertError)?;
+                    warn!(error = %error, "Error processing alert, skipping");
+                    // TODO: Handle alerts that we could not parse from avro
+                    // so we don't re-push them to the queue
```

Comment:
I'm curious if you've seen this happen with real data.

Reply:
yep, a couple of failed aren't parse successfully. Could be some edge case not handled by the custom avro parsing logic we have for ZTF. Something for me to investigate, planning on doing so tomorrow actually.

Reply:
That's really interesting. I'm eager to hear what you find out.